lib/twitter-ads/audiences/tailored_audience.rb

```diff
@@ -194,6 +215,49 @@ def self.memberships(account, params)
       [success_count, total_count]
     end
 
+    # This is a private API and requires whitelisting from Twitter.
+    #
+    # This endpoint will allow partners to add, update and remove users from a given tailored_audience_id.
+    # The endpoint will also accept multiple user identifier types per user as well.
+    #
+    # @example
+    #   tailored_audience.users(
+    #     account,
+    #     [
+    #       {
+    #         "operation_type": "Update",
+    #         "params": {
+    #           "effective_at": "2018-05-15T00:00:00Z",
+    #           "expires_at": "2019-01-01T07:00:00Z",
+    #           "users": [
+    #             {
+    #               "twitter_id": [
+    #                 "4798b8bbdcf6f2a52e527f46a3d7a7c9aefb541afda03af79c74809ecc6376f3"
+    #               ]
+    #             }
+    #           ]
+    #         }
+    #       }
+    #     ]
+    #   )
+    #
+    # @since 4.0
+    #
+    # @return success_count, total_count
+    def users(params)
```

Comment:
Please add `@param params [JSON object] A hash containing the list of users to be added/removed/updated` as a comment to describe the input format

Reply:
@tushdante 
Thank you for your comment. I will add it.